nntrainer/tensor/manager.cpp

```diff
@@ -254,10 +256,13 @@ std::vector<Weight *>
 Manager::requestWeights(const GraphNode &node,
                         const std::vector<Weight::Spec> &weights_spec) {
   const auto &exec_order = node.getExecutionOrder();
```

Comment:
You can later consider below for brevity
```cpp
const auto [forwarding, calcGradient, calcDerivateiv] = node.getExecutionOrder();
```

Reply:
I will add this in a separate PR.

Reply:
done.